Source/WebInspectorUI/UserInterface/Controllers/DOMManager.js

```diff
@@ -154,6 +154,49 @@ WI.DOMManager = class DOMManager extends WI.Object
         return Array.from(this._breakpointsForEventListeners.values());
     }
 
+    * attachedNodes(filter = null)
```

Comment:
Style: no space between the `*` and function name
Style: we normally prefer `{filter} = {}`